projects/fsastorefrontlib/src/core/my-account/facade/inbox.service.ts

```diff
@@ -19,6 +19,8 @@ export class InboxService {
 
   messagesSource = new BehaviorSubject<boolean>(false);
   messages = this.messagesSource.asObservable();
+  unreadMessagesStateSource = new BehaviorSubject<boolean>(false);
+  unreadMessagesState = this.unreadMessagesStateSource.asObservable();
```

Comment:
We should also start with the practice of marking these observables as in rest of application with `$` at the end.